UI/data/locale/en-US.ini

```diff
@@ -462,6 +462,22 @@ MissingFiles.AutoSearchText="OBS has found additional matches for missing files
 MissingFiles.NoMissing.Title="Missing Files Check"
 MissingFiles.NoMissing.Text="No files appear to be missing."
 
+# macOS permissions dialog
+MacPermissions.Title="Review App Permissions"
+MacPermissions.Description="OBS Studio requires your permission to be able to provide certain features. It is recommended to enable these permissions, but they are not required to use the app. You can always enable them later via the OBS Studio menu."
+MacPermissions.AccessGranted="Access Granted"
+MacPermissions.RequestAccess="Request Access"
+MacPermissions.OpenPreferences="Open %1 Preferences"
```

Comment:
This won't be a blocker for this PR, but I fear this translation key may need to be split up later as I am unsure if it'll work well in every language. Alternatively we could go the lazy way and just wrap the %1 with single quotes - though it might look a bit weird.